src/main/resources/tanagra/cms_synpuf/expanded/entity/device.json

```diff
@@ -30,25 +30,281 @@
           },
           "display" : "Healthcare Common Procedure Coding System (CMS)"
         },
-        "count" : 2828
+        "count" : 16
+      } ]
+    }
+  }, {
+    "type" : "SIMPLE",
```

Comment:
This happened because, due to the truncated dataset, there were less than 100 values for this property, so the expand config process generated more data.